packages/excalidraw/actions/actionTextAutoResize.ts

```diff
@@ -0,0 +1,48 @@
+import { isTextElement } from "../element";
+import { newElementWith } from "../element/mutateElement";
+import { measureText } from "../element/textElement";
+import { getSelectedElements } from "../scene";
+import { StoreAction } from "../store";
+import type { AppClassProperties } from "../types";
+import { getFontString } from "../utils";
+import { register } from "./register";
+
+export const actionUnwrapText = register({
```

Comment:
can you make the action name same as filename ? so both are in sync and it avoids any confusion as well

Reply:
my bad, forgot to rename action during file rename.